core/src/test/resources/application.properties-EXAMPLE

```diff
@@ -0,0 +1,9 @@
+# echache configuration properties for testing
```

Comment:
add slack channel name as a property please

Reply:
I think these are exclusive for unit tests so the slack channel isn't necessary